src/js/components/Table/Paginated/PaginatedTable.tsx

```diff
@@ -173,7 +175,7 @@ const PaginatedTable: PaginatedTableProps = ({
       setIsLoading(false);
       if (data) setDataSet(data);
 
-      if (currentTotalItems) setTotalItems(currentTotalItems);
+      if (typeof currentTotalItems === 'number') setTotalItems(currentTotalItems);
```

Comment:
Why is this change needed?

Reply:
We need to update the totalItems state even when the "currentTotalItems" is 0, before this change the if condition was returning false

Reply:
I thought on code setTotalItems(currentTotalItems || 0) but I don't know if this can be some side effects